README.md

```diff
@@ -68,22 +68,6 @@ To simplify the build process, these compiled files are checked-in to this repos
 npm run build
 ```
 
-## Contributor guidelines
+## CONTRIBUTING
```

Comment:
super nit: can you make this "Contributing" (noticing the case) just to match the style of the rest of the headers!

Reply:
Thanks for noticing @dnarcese ! I've removed the uppercase in the lattest commit. Let me know if there are any changes required!